pinot-core/src/main/java/org/apache/pinot/core/query/selection/SelectionOperatorUtils.java

```diff
@@ -568,4 +579,31 @@ private static Serializable getFormattedValue(Serializable value, DataSchema.Col
       queue.offer(value);
     }
   }
+
+  /**
+   * Helper class to compare rows.
+   */
```

Comment:
If I understand it correctly, the purpose of valueIndices is to record the index of order by column(s) in a row. Since this comparator is fed the entire row, we need to know which column is the order by column. If that is correct, can you please add this info in javadoc for anyone using this to understand what exactly is value indices>

Reply:
Good suggestion. Added javadoc